docs/concepts/crds/restoresession/index.md

```diff
@@ -285,7 +273,14 @@ Stash mounts an `emptyDir` for holding temporary files. It is also used for `cac
 
 #### spec.timeOut
 
-`spec.timeOut` specifies the maximum duration of the restore. `RestoreSession` will be considered `Failed` if the restore does not complete within this time limit. By default, Stash don't set any timeout for the restore.
+`spec.timeOut` specifies the maximum amount of time to wait for the restore to complete. If the restore doesn't complete within this time limit, Stash will mark the respective RestoreSession as `Failed`. You can specify the timeout in the following format:
+
+- Seconds `30s`
+- Minutes `10m`
+- Hours  `1h`
+- Combination of seconds, minutes hour 10m30s, `1h30m` etc.
```

Comment:
Combination of seconds, minutes hour 10m30s, `1h30m` etc. ->  Combination of seconds, minutes, and hours `10m30s`, `1h30m`  etc.